scripts/hostcfgd

```diff
@@ -223,6 +224,8 @@ class FeatureHandler(object):
         # Enable/disable the container service if the feature state was changed from its previous state.
         if self._cached_config[feature_name].state != feature.state:
             if self.update_feature_state(feature):
+                syslog.syslog(syslog.LOG_INFO, "Status of feature '{}' is changed from '{}' to '{}' ..."
+                          .format(feature_name, self._cached_config[feature_name].state, feature.state))
```

Comment:
Indentation issue at line 228?

Reply:
@yozhao101 what should be the coding style here? thought spliting one line should have more indent on 2nd line

Reply:
Like this: 

https://github.com/sonic-net/sonic-host-services/blob/f9af7aee3468786d941ac38ebdd4c9654f559ef6/scripts/hostcfgd#L218